src/lib/components/Swap/Price.tsx

```diff
@@ -0,0 +1,53 @@
+import { useLingui } from '@lingui/react'
+import { Trade } from '@uniswap/router-sdk'
+import { Currency, CurrencyAmount, Token, TradeType } from '@uniswap/sdk-core'
+import Row from 'lib/components/Row'
+import { ThemedText } from 'lib/theme'
+import formatLocaleNumber from 'lib/utils/formatLocaleNumber'
+import { useCallback, useMemo, useState } from 'react'
+import { formatCurrencyAmount, formatPrice } from 'utils/formatCurrencyAmount'
+
+import { TextButton } from '../Button'
+
+interface PriceProps {
+  trade: Trade<Currency, Currency, TradeType>
+  quoteUSDC?: CurrencyAmount<Token>
```

Comment:
Could we just add a comment explaining what `quoteUSDC` is? Took me a second to parse everything

Reply:
Renamed to `outputUSDC` to clarify this.